### PR TITLE
Fix derivation with images in object descriptions

### PR DESCRIPTION
**Pull request by arnauddieumegard**
Functional modifications are in files: 
- plugins/org.polarsys.capella.filtering.tools/src/org/polarsys/capella/filtering/tools/view/DiagCriteriaVisibilityView.java for dialog corrections
- plugins/org.polarsys.capella.filtering.tools/src/org/polarsys/capella/filtering/tools/actions/FilteringExtractionJob.java for derivation corrections

Other modifications are for releng and test migration

### CODE REVIEW

File: plugins/org.polarsys.capella.filtering.tools/src/org/polarsys/capella/filtering/tools/view/DiagCriteriaVisibilityView.java [hunk 1]
```diff
@@ -282,10 +284,10 @@ public void widgetSelected(SelectionEvent e) {
             }
             // Clear filtering global cache for current project
             if (treeViewerInput instanceof EObject) {
-              Project project = CapellaProjectHelper.getProject((EObject) treeViewerInput);
-              FilteringToolsPlugin.getGlobalFilteringCache().remove(project);
+                Project project = CapellaProjectHelper.getProject((EObject) treeViewerInput);
+                FilteringResult allCriteriaResult = FilteringFactory.eINSTANCE.createFilteringResult();
```
[the remark this answers is not in the document]
Comment (arnauddieumegard) [reply plugins/org.polarsys.capella.filtering.tools/src/org/polarsys/capella/filtering/tools/view/DiagCriteriaVisibilityView.java:288]: Sure

**Comment by minhtutonthat** [comment]
Since we now have a Capella with the expected API, could we merge this PR?

**Comment by arnauddieumegard** [comment]
It should now be good, I will either need a Capella stable or to change the TP to link to the nightly.

**Comment by minhtutonthat** [comment]
The current Filtering TP already depends on Capella nightlly (https://github.com/eclipse/capella-filtering/blob/master/releng/org.polarsys.capella.filtering.update/capella.target-definition.targetplatform), doesn't it?

**Comment by arnauddieumegard** [comment]
Now this is good 👍

**Comment by arnauddieumegard** [comment]
If you can wait some more minutes, I will fix the Code Smells.

**Comment by arnauddieumegard** [comment]
Now we are good to go !

**Review by minhtutonthat — APPROVED** [review]

**Pull request merged**